### ISSUE TITLE
https://github.com/kelaberetiv/TagUI/issues/86#issue-303370944

### ISSUE THREAD

**Issue by pauljohnfox** [kelaberetiv/TagUI#858]
computer:clamav-0.102.4 myusername$ brew upgrade valgrind
==> Upgrading 1 outdated package:
valgrind 3.15.0 -> 3.16.1
==> Upgrading valgrind 3.15.0 -> 3.16.1 
valgrind: This formula either does not compile or function as expected on macOS
versions newer than High Sierra due to an upstream incompatibility.
Error: valgrind: An unsatisfied requirement failed this build.
computer:clamav-0.102.4 myusername$ 

This is relative to the issue that has been marked "resolved," but absolutely NONE of the "resolutions," work for me on Mojave. It'll compile and it'll install, but I always end up being pointed at the wrong library and, obviously, it can't load. 

dyld: Library not loaded: /opt/local/lib/libssl.1.0.0.dylib
  Referenced from: /usr/local/bin/psql
  Reason: image not found

freshclam
dyld: Library not loaded: /usr/local/opt/openssl/lib/libssl.1.0.0.dylib
  Referenced from: /usr/local/bin/freshclam
  Reason: image not found

ls -l /usr/local/opt/openssl*
lrwxr-xr-x  1 myusername  admin  28 Aug 22 16:38 /usr/local/opt/openssl -> ../Cellar/openssl@1.1/1.1.1g
lrwxr-xr-x  1 myusername  admin  28 Aug 22 16:38 /usr/local/opt/openssl@1.1 -> ../Cellar/openssl@1.1/1.1.1g

The configure command I'm trying to use:

./configure --with-openssl=/usr/local/opt/openssl --with-libjson=yes --enable-check CPPFLAGS="-I/usr/local/opt/openssl/include" LDFLAGS="-L/usr/local/opt/openssl/lib/" --prefix=$HOME/clamav --disable-clamav

Summary: 

Summary of detected features follows
              OS          : darwin18.7.0
              pthreads    : yes ()
configure: Summary of miscellaneous features
              check       : -lcheck -L/usr/local/opt/openssl/lib/
              fanotify    : no (disabled)
              fdpassing   : 1
              IPv6        : yes
              openssl     : /usr/local/opt/openssl
              libcurl     : /usr
configure: Summary of optional tools
              clamdtop    : yes (-lncurses)
              milter      : no (missing libmilter) (disabled)
              clamsubmit  : yes (libjson-c-dev found at /usr/local, linking=dynamic)
              clamonacc   : no (auto)
configure: Summary of engine performance features
              release mode: yes
              llvm        : no (disabled)
              mempool     : yes
configure: Summary of engine detection features
              iconv       : yes
              bzip2       : ok
              zlib        : yes (from system)
              unrar       : yes
              preclass    : yes (libjson-c-dev found at /usr/local, linking=dynamic)
              pcre        : /usr/local/Cellar/pcre2/10.35
              libmspack   : yes (Internal)
              libxml2     : yes, from /usr
              yara        : yes
              fts         : yes (libc)
configure: WARNING:
****** libjson-c is known to share symbol names with other JSON libraries
****** which may result in crashes for applications that use libclamav.
****** Consider using --with-libjson-static=path/to/libjson-c.a,
****** providing a json-c library that was compiled with CFLAGS="-fPIC".

brew upgrade
==> Upgrading 2 outdated packages:
check 0.12.0 -> 0.15.2
valgrind 3.15.0 -> 3.16.1
==> Upgrading check 0.12.0 -> 0.15.2 
==> Downloading https://homebrew.bintray.com/bottles/check-0.15.2.mojave.bottle.tar.gz
######################################################################## 100.0%
==> Pouring check-0.15.2.mojave.bottle.tar.gz
🍺  /usr/local/Cellar/check/0.15.2: 42 files, 533KB
Removing: /usr/local/Cellar/check/0.12.0... (42 files, 521.9KB)
Removing: /Users/myusername/Library/Caches/Homebrew/check--0.12.0.high_sierra.bottle.tar.gz... (145.2KB)
==> Upgrading valgrind 3.15.0 -> 3.16.1 
valgrind: This formula either does not compile or function as expected on macOS
versions newer than High Sierra due to an upstream incompatibility.
Error: valgrind: An unsatisfied requirement failed this build.

I'm not "upgrading," to Catalina, and I need Mojave for audio production features I use so I'm stuck with Mojave. 

I have tried disabling SIP, escaping the @ sumbol, and at least a dozen other "solutions," that didn't get me anywhere closer to being able to virus check my system. If I didn't already believe Apple is intentionally making things impossible, I'd certainly suspect so now. 

Is there ANY solution for this on Mojave now? If you have one, please treat me like I'm stupid and spell out the steps. I won't be offended and I'll be very grateful, and I suspect it'll hep others as well. 

Thank you.

**Comment by pauljohnfox** [comment]
I am still having problems getting this working and there doesn't seem to be any end in sight. 

> freshclam
dyld: Library not loaded: /usr/local/opt/openssl/lib/libssl.1.0.0.dylib
  Referenced from: /usr/local/bin/freshclam
  Reason: image not found
Abort trap: 6

Absolutely nothing is working as a solution. I've tried "switching," I've tried turning off SIP, I've tried reinstalling the whole damn thing. Nothing. I've tried ignore dependencies also to no avail. 

Is ClamAV just straight up broken for some of us?
